src/commands/ping.json

```diff
@@ -17,6 +17,18 @@
             "REQUEST_POLICY:ALL_SHARDS",
             "RESPONSE_POLICY:ALL_SUCCEEDED"
         ],
+        "reply_schema": {
+            "anyOf": [
```

Comment:
Should this be `oneOf` ?. Otherwise, if reply is `POOOONG`, it will match the second schema.

Reply:
`POOOONG ` is a legal reply for PING

Reply:
oh, I didn't know that